### ISSUE TITLE
Restore mempool fork

### ISSUE THREAD

**Issue by runbtc** [bisq-network/proposals#454]
> _This is a Bisq Network proposal. Please familiarize yourself with the [submission and review process](https://bisq.wiki/Proposals)._

# Summary
Since [Bisq has been removed](https://github.com/mempool/mempool/pull/4786) from the mempool project, we will now need to maintain our [fork](https://github.com/bisq-network/mempool) which was archived after a [PR](https://github.com/bisq-network/mempool/pull/1) to merge in the latest upstream changes stalled.

Along with [some changes I made to the install scripts](https://github.com/mempool/mempool/pull/5058), I was able to successfully deploy an instance from the latest commit in that PR, and is currently running at http://runbtcx3wfygbq2wdde6qzjnpyrqn3gvbks7t5jdymmunxttdvvttpyd.onion. So there should not be a lot of work to get the fork up to date and in a functional state.

# Action Items
- [x] Restore the [forked repository](https://github.com/bisq-network/mempool)
- [x] Open a new PR up to prior to [Bisq being removed](https://github.com/mempool/mempool/commit/83dbae3a70dc42be8b953a1ab666bcabbf2a3d13)
- [x] Merge [install script changes](https://github.com/mempool/mempool/pull/5058) [@runbtc]
- [ ] Update/deploy a mempool instance from the fork [@runbtc]

# Future Maintenance
Going forward, we will need to maintain the fork on an as-needed basis. Mainly if any worth while functionality or bug fixes are implemented upstream. I am not able to commit to taking ownership of this, but will likely be able to help as necessary.

**Comment by clearwater-trust** [comment]
301 redirect from [bisq.info](https://bisq.info)

Thank you.

**Comment by alvasw** [comment]
Nevermind!

**Comment by HenrikJannsen** [comment]
Any update on that project?

**Comment by runbtc** [comment]
Looks like all is done except deploying a node from this fork. I will do that in the coming days.

**Comment by runbtc** [comment]
Just an update. I deployed an instance, though ran into a few issues along the way for which I opened a couple PR's. I have a few more changes pending.

**Comment by runbtc** [comment]
I consider this proposal complete - the fork has been restored. The only outstanding action item was to update/deploy a mempool instance from the fork but I don't think is necessary to complete this proposal. I do still plan on updating my instance once https://github.com/bisq-network/mempool/pull/20 is complete.

**Comment by pazza83** [comment]
Will there be a clear net instance?

**Comment by runbtc** [comment]
I was not planning on a clear net instance. But if necessary, I may be able to configure access via IP address only - no DNS record.

**Comment by pazza83** [comment]
I think a clear net instance would be good that can be included in the Bisq app. The current ones are a little dated. There is only one BSQ explorer that does not always work.
